packages/core/src/repository/query/network-query.ts

```diff
@@ -0,0 +1,20 @@
+import { ParameterType } from '../../data';
+
+export abstract class NetworkQuery {
+    constructor(public readonly endpoint: string) {}
+
+    get body(): string | FormData {
```

Comment:
**Nit:** Is there a reason of using getter methods instead of regular methods? Any advantage over `getBody/getUrlParameters/getQueryParameters`?

Reply:
I could ask the same the other way around... These are "funtions" that return a structure based on the attributes, so no actions and no input parameters, so they are dynamic attributes.